crates/docblock/src/tag.rs

```diff
@@ -570,6 +579,27 @@ pub fn parse_import_type_tag(content: &str, span: Span) -> Option<ImportTypeTag>
     Some(ImportTypeTag { span, name: name.to_owned(), from: imported_from.to_owned(), alias })
 }
 
+pub fn parse_property_tag(content: &str, span: Span) -> Option<PropertyTag> {
+    let (type_string, rest_slice) = split_tag_content(content, span)?;
+
+    // Type must exist and be valid
+    if type_string.value.is_empty()
+        || type_string.value.starts_with('{')
+        || (type_string.value.starts_with('$') && type_string.value != "$this")
+    {
+        return None;
+    }
+
+    if rest_slice.is_empty() {
+        return None;
+    }
+
+    let var_part = rest_slice.split_whitespace().next()?;
+    let variable = parse_var_ident(var_part)?;
+
+    Some(PropertyTag { span, type_string, variable, is_read: false, is_write: false })
+}
+
```

Comment:
this will return `None` for `@property $foo`. we should in this case set `TypeString` to `None`.